ext/jenkins/lein-test.sh

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
+
+set -e
+set -x
+
+DBNAME=$(echo "puppetdb_${PUPPETDB_BRANCH}_${BUILD_ID}_${BUILD_NUMBER}_${PUPPETDB_DBTYPE}_${JDK}" | tr - _)
+DBUSER="puppetdb"
+DBHOST="fixture-pg94.delivery.puppetlabs.net"
```

Comment:
They will have different `BUILD_ID`/`BUILD_NUMBER` values and thus have different `DBNAME`